Cargo.toml

```diff
@@ -1,8 +1,19 @@
 [package]
+# NOTE: 2018 is needed else "maybe a missing crate `core`"
+edition = "2021"
 name = "color_quant"
 license = "MIT"
 version = "1.1.0"
 authors = ["nwin <nwin@users.noreply.github.com>"]
 readme = "README.md"
 description = "Color quantization library to reduce n colors to 256 colors."
 repository = "https://github.com/image-rs/color_quant.git"
+rust-version = "1.56"
+
+[dependencies]
+num-traits = { version = "0.2", default-features = false, optional = true }
+
+[features]
+default = ["std"]
+std = []
+num-traits = ["dep:num-traits"]
```

Comment:
Optional dependencies are usable as features, directly. This syntax is only necessary if they need to be disambiguated. In this case there is no such need, i.e this line can be removed. No other changes should be necessary.

Reply:
Oh? I did not know that. Thx!